internal/ssm_test.go

```diff
@@ -31,13 +36,18 @@ func TestGetSSMParameter(t *testing.T) {
 	// Setup Test
 	mockSvc := &mockSSMClient{}
 
-	for k, v := range ssmTestValues {
-		t.Logf("Key: %s should have value: %s", k, v.expectedValue)
-		value, err := getSSMParameter(mockSvc, v.path, v.required, v.decrypt)
-		if err != nil {
-			t.Errorf("Expected %s , got %s", v.expectedValue, err)
-		} else if value != v.expectedValue {
-			t.Errorf("Expected %s , got %s", v.expectedValue, value)
+	for k, v := range fakeSSMStore {
+		expectedValueStr := "nil"
+		if v.expectedValue != nil {
+			expectedValueStr = *v.expectedValue
```

Comment:
shouldn't this be an Assert?

Reply:
No, the value can be nil when missing, its just to make the pretty print.